src/components/views/rooms/RoomSettings.js

```diff
@@ -252,6 +270,30 @@ module.exports = React.createClass({
         });
     },
     
+    _onRoomAccessRadioToggle: function(ev) {
+        var self = this;
```

Comment:
No closures are being used here at all; using `self` in this function is redundant.